JS.md

```diff
@@ -522,12 +520,8 @@ or `null`, throw a [`TypeError`](https://tc39.github.io/ecma262/#sec-native-erro
 
 Let `i` be the result of [`ToNonWrappingUint32`](#tononwrappinguint32)(`index`).
 
-If `v` is an [Exported Function Exotic Object](#exported-function-exotic-objects):
-* Set the `i`th element of `T.[[Table]]` to the `v.[[FunctionIndex]]`th function
-  in the module's [function index space](Modules.md#function-index-space).
```

Comment:
Yes, but when we allowed tables to be im/exported the spec's representation had to change, and table elements are now of type `closure`, as defined here:
https://github.com/WebAssembly/spec/blob/master/ml-proto/spec/instance.ml#L11
This is pretty much the spec's rendering of the [[Values]] array. The index is gone, we need to find some language saying how JS functions map to the spec's "host functions" to make the connection. And at that point we are essentially just duplicating the information, I think.